flocker/provision/_common.py

```diff
@@ -22,3 +22,18 @@ class PackageSource(object):
     :ivar bytes build_server: The builderver to install from.
         Only meaningful if a branch is specified.
     """
+
+
+@attributes(
+    ['version', 'release', 'distribution', 'architecture']
+)
+class Kernel(object):
+    """
+    Represents the version information for a kernel package.
```

Comment:
This seems at least slightly RPM specific. Perhaps that should be mentioned here.